### DISCUSSION TITLE
PnPMicrosoft365Group and Site collection association

### DISCUSSION THREAD

**Discussion by Ofer-Gal** [pnp/powershell#2957] [Q&A]
I have a bad situation. I had a team site with 365 group and some how the 365 group was deleted.
Now when I run:
`Add-PnPMicrosoft365GroupToSite -Url $theSite -Alias "1579.028" ...` 
I get: `This site already has an O365 Group attached.`  and I can see $site.RelatedGroupId==e378ae1b-bffb-465e-bc0a-df8eac647f43
How can I disjoint (divorce) the Site from the none existing 365 group so I can relate another group to it? (I tried -Force no go )

Can anyone help me?

Thanks in advance

**Comment by AndersRask** [comment]
Did you try to restore the deleted group? This would probably be your best bet.
Deleting the group would normally trigger the deletion of the site too, so not sure how you managed to get your site orphaned :) 
If restoring does not work, I would open a ticket to MS Support

```
$group = Get-PnPDeletedMicrosoft365Group -Identity e378ae1b-bffb-465e-bc0a-df8eac647f43
Restore-PnPDeletedMicrosoft365Group -Identity $group
```

**Reply by Ofer-Gal** [reply]
It is no longer in the Deleted Groups. I tried that.
Wonder if I change the template to "site with no group" it will let me "Connect to new Microsoft 365 Group"
